TDOAuth.podspec

```
@@ -25,17 +25,13 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '3.0'
   s.osx.deployment_target = '10.10'
 
-  s.swift_versions = [ '4.0', '4.2', '5.0' ]
+  s.swift_versions = [ '4.0', '4.2', '5.0', '5.1', '5.2', '5.3', '5.4', '5.5', '5.6' ]
 
-  s.default_subspec = 'ObjC'
+  s.default_subspec = 'Swift'
 
   s.subspec 'Swift' do |ss|
-    ss.source_files = 'Source/*.swift'
+    ss.source_files = 'Source/*.swift', 'Source/compat/*.{swift}'
```

Should this be all inclusive and include TDOAuth1.swift ?
```suggestion
    ss.source_files = 'Source/*.swift', 'Source/**/*.{swift}'
```